src/Razor/Razor/src/RenderAsyncDelegate.cs

```diff
@@ -3,7 +3,7 @@
 
 using System.Threading.Tasks;
 
-namespace Microsoft.AspNetCore.Mvc.Razor
+namespace Microsoft.AspNetCore.Razor
```

Comment:
I don't think we can make this breaking change, I don't think we can predict the impact of changing this.